src/ProjectTemplates/test/MvcTemplateTest.cs

```diff
@@ -33,11 +33,11 @@ public ITestOutputHelper Output
     public async Task MvcTemplate_NoAuthFSharp() => await MvcTemplateCore(languageOverride: "F#");
 
     [ConditionalFact]
-    [SkipOnHelix("Cert failure, https://github.com/dotnet/aspnetcore/issues/28090", Queues = "All.OSX;" + HelixConstants.Windows10Arm64 + HelixConstants.DebianArm64)]
+    [SkipOnHelix("Cert failure, https://github.com/dotnet/aspnetcore/issues/28090", Queues = "All.OSX;" + "All.Ubuntu;" + HelixConstants.Windows10Arm64)]
```

Comment:
This doesn't look right because it will _enable_ the tests on Debian ARM64 again. Why skip on Ubuntu at all ❔

suggest
```suggestion
    [SkipOnHelix("Cert failure, https://github.com/dotnet/aspnetcore/issues/28090", Queues = "All.OSX;Debian.11.Amd64;Debian.11.Amd64.Open;" + HelixConstants.Windows10Arm64 + HelixConstants.DebianArm64)]
```

On the other hand, _any_ use of `HelixConstants` will break internal PRs / any internal testing on those queues.

Reply:
> On the other hand, any use of HelixConstants will break internal PRs / any internal testing on those queues.

Why is that the case? We use HelixConstants in a bunch of places, wouldn't internal PRs already be broken?

Reply:
> Why is that the case?

`HelixConstants` uses "Blah.Open" names and those shouldn't match the `helix` environment variable because it won't include `.Open`. I'm no longer sure how the constants work anywhere. Must be missing something&hellip;

Reply:
Oh right: We don't have an internal `helix-matrix` queue anymore. Nothing listed in `HelixConstants` relates to a queue we use in `aspnetcore-ci[-official]` _ever_